lib/foreman_openscap/engine.rb

```diff
@@ -32,41 +32,50 @@ class Engine < ::Rails::Engine
-                                         :scaptimony_policies => [:index, :show],
+          permission :view_arf_reports, {:scaptimony_arf_reports   => [:index, :show, :parse, :auto_complete_search],
+                                         :scaptimony_policies      => [:index, :show],
+                                         :scaptimony_dashboard     => [:index],
+                                         :scaptimony_hosts         => [:show],
                                          :scaptimony_scap_contents => [:index, :show],
-                                        }
-          permission :edit_compliance, {:scaptimony_arf_reports => [:destroy],
-                                        :scaptimony_policies => [:new, :create, :edit, :update, :destroy,
-                                          :select_multiple_hosts, :upldate_multiple_hosts],
-                                        :scaptimony_scap_contents => [:new, :create, :edit, :update]
-                                       }
+                                      }
+          permission :edit_compliance, {:scaptimony_arf_reports           => [:destroy],
+                                        :scaptimony_policies              => [:new, :create, :edit, :update, :destroy, :parse,
+                                                                              :select_multiple_hosts, :update_multiple_hosts,
+                                                                              :auto_complete_search, :scap_content_selected,
+                                                                              :disassociate_multiple_hosts, :remove_policy_from_multiple_hosts],
+                                        :scaptimony_scap_contents         => [:new, :create, :edit, :update, :destroy, :auto_complete_search],
+                                        'api/v2/compliance/policies'      => [:create, :update, :destroy, :show, :index, :content],
+                                        'api/v2/compliance/scap_contents' => [:create, :update, :destroy, :show, :index],
+                                        'api/v2/compliance/arf_reports'   => [:index, :show, :destroy]
+                                     }
+          permission :create_arf_report, {'api/v2/compliance/arf_reports' => [:create]}
         end
 
         role "View compliance reports", [:view_arf_reports]
         role "Edit compliance policies", [:edit_compliance]
+        role "Create ARF report", [:create_arf_report]
 
         #add menu entries
         divider :top_menu, :caption => N_('Compliance'), :parent => :hosts_menu
         menu :top_menu, :compliance_policies, :caption => N_('Policies'),
-             :url_hash => {:controller => :'scaptimony_policies', :action => :index },
-             :parent => :hosts_menu
+             :url_hash                                 => {:controller => :'scaptimony_policies', :action => :index},
```

Comment:
if slightly possible, don't indent like in puppet dsl, any change to one line usually means changes to surrounding lines just because of indentation